### PR TITLE
feat: Make headers sticky on Members page

### LINKED ISSUE #16293: [Feature]: Make Table and Tab headers sticky
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

Today, Table and Tab headers are not always sticky. Making them sticky would provide a better user experience.

### Why should this be worked on?

Making the headers sticky would provide a better user experience.

### PR DESCRIPTION
**Pull request by ankitakinger**
## Description

> Implemented sticky header on the Members page to provide better user experience.

Fixes #16293 

## Type of change

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

> Tested the Members page containing short and long lists of members. Tested its scroll behavior and tested its dropdown behavior.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by berzerkeer — APPROVED** [review]

File: app/client/src/pages/Settings/components.tsx [hunk 1]
```diff
@@ -36,34 +34,12 @@ export const SettingsFormWrapper = styled.div`
   }
 `;
 
+export const MaxWidthWrapper = styled.div`
+  max-width: 40rem;
```
Comment (IAmAnubhavSaini) [inline app/client/src/pages/Settings/components.tsx:38]: Do we need it in `rem`? We are mostly using pixels or percentages.

**Review by IAmAnubhavSaini — APPROVED** [review]
PR Looks good. We really need to move away from styled-components though. In general, in all of the app, the styling related code is becoming complex and almost unreadable.

**Pull request merged**